Automattic/jetpack
# Beta: Adding trunk branch cases

#### Changes proposed in this Pull Request:

* In preparation for renaming 'master' to 'trunk' in the Jetpack monorepo and mirror repos, this PR adds a trunk branch case to the Beta Tester plugin files. This should allow for a seemless change when the 'master' branch is renamed.

#### Jetpack product discussion

p9dueE-59T-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Testing locally with this PR checked out, does everything work properly with the Beta plugin installed and active (nothing appears broken?) - try activating different options - RC, bleeding edge, latest stable. Try for different plugins installed,  eg. Jetpack Backup.
* To test out some of the changes (to attempt to replicate a plugin's default branch being 'trunk'), you can also try the following:
  * In `/admin/branch-card.template.php`, before the first `if` statement in the `$tmp` declared function, add the following:
	```
	if ( 'master' === $branch->source ) {
		$branch->source = 'trunk';
	}
   * Then hover over the 'activate' button for 'bleeding edge' and the URL should show trunk instead of master.
   * Clicking it should result in an error message saying 'No trunk build is available for xxx'
* If you are aware of any other ways to test this out, please do and let me know what you've tried or would suggest, too.
